data/mods/Aftershock/modinfo.json

```
@@ -4,7 +4,7 @@
     "ident": "aftershock",
     "name": "Aftershock",
     "authors": [ "Esther", "Coolthulu" ],
-    "maintainers": [ "Esther" ],
+    "maintainers": [ "Esther", "Maleclypse" ],
```

They aren't around anymore.
```suggestion
    "maintainers": [ "Maleclypse" ],
```

---

I wasn’t sure exactly how long they’d been gone so I didn’t want to be premature in removing them. I’ll go ahead and remove.